specification/synapse/data-plane/Microsoft.Synapse/preview/2021-06-01-preview/runNotebook.json

```diff
@@ -0,0 +1,359 @@
+      "get": {
+        "tags": [
+          "RunNotebook"
+        ],
+        "operationId": "GET_RunNotebookStatus",
+        "x-ms-examples": {
+          "Get Run Notebook Status": {
+            "$ref": "./examples/RunNotebook_GetStatus.json"
+          }
+        },
+        "description": "Get RunNotebook Status for run id.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/VersionParameter"
+          },
+          {
+            "$ref": "#/parameters/RunId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/RunNotebookResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "./artifacts.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/runNotebookApi/versions/{version}/run/{runId}/cancel": {
+      "post": {
+        "tags": [
+          "RunNotebook"
+        ],
+        "operationId": "Cancel_RunNotebook",
+        "x-ms-examples": {
+          "Cancel Run Notebook": {
+            "$ref": "./examples/RunNotebook_Cancel.json"
+          }
+        },
+        "description": "Cancel notebook run.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/VersionParameter"
+          },
+          {
+            "$ref": "#/parameters/RunId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/RunNotebookResponse"
+            }
+          },
+          "408": {
+            "description": "RequestTimeout.",
+            "x-ms-error-response": true,
+            "schema": {
+              "$ref": "#/definitions/RunNotebookResponse"
+            }
+          },
+          "default": {
+            "description": "An error response received from the Azure Synapse service.",
+            "schema": {
+              "$ref": "./artifacts.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/runNotebookApi/versions/{version}/run/{runId}/snapshot": {
+      "get": {
+        "tags": [
+          "RunNotebook"
+        ],
+        "operationId": "Get_RunNotebookSnapshot",
+        "x-ms-examples": {
+          "Get Run Notebook Snapshot": {
+            "$ref": "./examples/RunNotebook_GetSnapshot.json"
+          }
+        },
+        "description": "Get RunNotebook Snapshot for run id.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/VersionParameter"
+          },
+          {
+            "$ref": "#/parameters/RunId"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Ok.",
+            "schema": {
+              "$ref": "#/definitions/RunNotebookResponse"
```

Comment:
GetSnapshot doesn't return `RunNotebookResponse`. It returns something contains updated notebook content.